src/internal/clusterstate/v2.8.0/pfsdb.go

```diff
@@ -422,20 +415,60 @@ func migratePage(ctx context.Context, tx *pachsql.Tx, page []commitCollection) e
 		if !commit.StartTime.Valid {
 			return errors.Errorf("commit %s has a nil start time", commit.IntID)
 		}
-		_, err = tx.NamedExecContext(ctx, migrateQuery, commit)
+		err = batcher.Add(migrateCommit(commit))
 		if err != nil {
-			return errors.Wrap(err, fmt.Sprintf("migrating commit %d", commit.IntID))
+			return errors.Wrap(err, "batching commit migration query")
 		}
 		if ancestry.ParentCommit == "" && ancestry.ChildCommits == nil {
 			continue
 		}
-		if err := migrateRelatives(ctx, tx, commit.IntID, ancestry); err != nil {
-			return errors.Wrap(err, fmt.Sprintf("migrating relatives of commit %d", commit.IntID))
+		err = batcher.Add(migrateRelatives(commit.IntID, ancestry))
+		if err != nil {
+			return errors.Wrap(err, "batching commit ancestry query")
 		}
 	}
 	return nil
 }
 
+func migrateCommit(commit *Commit) string {
+	migrateQuery := `WITH repo_row_id AS (SELECT id from pfs.repos WHERE name='%s' AND type='%s' 
+                                                AND project_id=(SELECT id from core.projects WHERE name='%s'))
+		UPDATE pfs.commits SET 
+			commit_id='%s', 
+			commit_set_id='%s',
+		    repo_id=(SELECT id from repo_row_id), 
+		    branch_id=(SELECT id from pfs.branches WHERE name='%s' AND repo_id=(SELECT id from repo_row_id)), 
+			description='%s', 
```

Comment:
I don't think this is sufficient to sanitize description.  As a general rule: an escaping procedure has to iterate over the string and replace characters.  Can we find a function for this in the standard library or the postgres driver?